vendor/engines/projects/app/views/projects/projects/new.html.haml

```diff
@@ -0,0 +1,12 @@
+= render "shared/headers/ckeditor"
+
+= content_for :h1 do
+  = t(".head")
+
+= simple_form_for @project, url: facility_projects_path(current_facility), method: :post do |f|
+  .form-inputs
+    = f.input :name, hint: t(".name")
```

Comment:
Not a big deal, but I like to use a `hints` prefix, so this would be `.hints.name`. You could also put the hints in the (`simple_form.hints` space)[https://github.com/plataformatec/simple_form#i18n] and have it automatically picked up. Currently we have a few things in the `simple_form.yml` file using that technique, but I've become less of a fan using that file because it separates the hints from other translations related to the model. But it might be fine to use within the engine file. I don't have a strong preference.